index.html

```diff
@@ -886,7 +901,136 @@ <h4 id="event-handlers"><span class="secno">6.3.4 </span>
         </section>
       </section>
       <section>
-        <h3 id="interface-navigatorpresentation"><span class="secno">6.4 </span>
+        <h3 id="interface-availability"><span class="secno">6.4 </span>
+          Interface <a href="#availability"><code>Availability</code></a>
+        </h3>
+        <pre class="idl">interface <dfn id="availability">Availability</dfn> : EventTarget {
+  readonly attribute boolean current;
```

Comment:
I don't know if ```current``` is the best name. I would consider ```value``` but it doesn't matter much I guess.